utils/fetchProfile.ts

```diff
@@ -121,3 +126,40 @@ const createProfileObject = async (
     issuedAssetAddresses,
   }
 }
+
+/**
+ * Get profile from RPC
+ *
+ * @param profileAddress
+ * @returns
+ */
+const getProfileData = async (profileAddress: Address) => {
+  const { getInstance } = useErc725()
+  const erc725 = getInstance(
+    profileAddress,
+    LSP3ProfileMetadataSchema as ERC725JSONSchema[]
+  )
+  const profileMetadata = await erc725.fetchData('LSP3Profile')
+
+  // NOTE: Some profiles have been encoded with keccak256(bytes) instead of keccak256(utf8)
+  // This little trick allows a smooth transition to the new encoding
+  try {
+    if (
+      Object.prototype.toString.call(profileMetadata.value) ===
```

Comment:
But this automatically should happen with the new version of erc725. Are you ever hitting this code?

Reply:
If this is fixed in erc725 already then I guess we don't need this code, thanks for pointing this out.